docs/source/example_notebooks/graph_conditional_independence_refuter.ipynb

```diff
@@ -176,7 +255,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "refuter_object = model.refute_graph(k=2, method_name = \"partial_correlation\")  #Change k parameter to test conditional independence given different number of variables "
+    "refuter_object = model.refute_graph(k=1, independence_test = {'test_for_continuous': 'partial_correlation', 'test_for_discrete' : 'conditional_mutual_information'}) #Change k parameter to test conditional independence given different number of variables "
```

Comment:
are there examples of inputs that you give that include 
1) CD|D
2) CC|D
3) DD|C
4) CD|C
where C stands for cont variable and D for discrete variable. All four are interesting cases and should be discussed in the notebook.